components/camel-bean/src/main/java/org/apache/camel/component/bean/BeanComponent.java

```diff
@@ -36,9 +37,19 @@
     @SuppressWarnings("unchecked")
     private final Map<BeanInfoCacheKey, BeanInfo> beanInfoCache = LRUCacheFactory.newLRUSoftCache(1000);
 
-    @Metadata(label = "advanced", description = "If enabled, Camel will cache the result of the first Registry look-up."
-        + " Cache can be enabled if the bean in the Registry is defined as a singleton scope.")
+    @Deprecated
+    @Metadata(defaultValue = "true", description = "Use singleton option instead.")
     private Boolean cache;
+    @Metadata(defaultValue = "Singleton", description = "Scope of bean."
+            + " When using singleton scope (default) the bean is created or looked up only once and reused for the lifetime of the endpoint."
+            + " The bean should be thread-safe in case concurrent threads is calling the bean at the same time."
+            + " When using request scope the bean is created or looked up once per request (exchange). This can be used if you want to store state on a bean"
+            + " while processing a request and you want to call the same bean instance multiple times while processing the request."
+            + " The bean should not be thread-safe as the instance is only called from the same request.  "
```

Comment:
Maybe `The bean does not have to be thread-safe` would be clearer (as it is o.k. for the bean to be thread-safe)? What do you think?

Reply:
Yeah well spotted, thanks